build fails on zig 0.9.1 due to imgui build.zig

Just cloned the repo and ran `zig build help`

```
$ zig build help
./gamekit/deps/imgui/build.zig:70:9: error: no member named 'source' in struct 'std.build.Pkg'
        .source = .{ .path = prefix_path ++ "gamekit/deps/imgui/imgui.zig" },
        ^
./build.zig:110:52: note: called from here
    const imgui_pkg = imgui_builder.getImGuiPackage(prefix_path);
```

looks like changing `.source` to `.path` fixes the problem

--- a/build.zig
+++ b/build.zig
@@ -6,7 +6,7 @@ pub const ShaderCompileStep = @import("shader_compiler/shader_compiler.zig").Sha
 pub fn getRenderKitPackage(comptime prefix_path: []const u8) Pkg {
     return .{
         .name = "renderkit",
-        .path = .{ .path = prefix_path ++ "renderkit/renderkit.zig" },
+        .source = .{ .path = prefix_path ++ "renderkit/renderkit.zig" },
     };
 }
 

--- a/shader_compiler/shader_compiler.zig
+++ b/shader_compiler/shader_compiler.zig
@@ -279,7 +279,7 @@ pub const ShaderCompileStep = struct {
             return;
         };
 
-        var dir_obj = try std.fs.cwd().openDir(self.shader_out_path[0 .. self.shader_out_path.len - 1], .{ .iterate = true });
+        var dir_obj = try std.fs.cwd().openIterableDir(self.shader_out_path[0 .. self.shader_out_path.len - 1], .{});
         defer dir_obj.close();
         var walker = try dir_obj.walk(self.builder.allocator);
         defer walker.deinit();
@@ -305,7 +305,7 @@ pub const ShaderCompileStep = struct {
 
                 if (uses_default_vs) {
                     // we may have a relative path if the Options were given a relative output path for the shader or package
-                    if (std.fs.path.isAbsolute(entry.path)) try std.fs.deleteFileAbsolute(entry.path) else try dir_obj.deleteFile(entry.path);
+                    if (std.fs.path.isAbsolute(entry.path)) try std.fs.deleteFileAbsolute(entry.path) else try dir_obj.dir.deleteFile(entry.path);
                 }
             }
         }

--- a/shader_compiler/shader_compiler.zig
+++ b/shader_compiler/shader_compiler.zig
@@ -377,8 +377,7 @@ pub const ShaderCompileStep = struct {
         max_output_bytes: usize = 50 * 1024,
         expand_arg0: std.ChildProcess.Arg0Expand = .no_expand,
     }) !std.ChildProcess.ExecResult {
-        const child = try std.ChildProcess.init(args.argv, args.allocator);
-        defer child.deinit();
+        var child = std.ChildProcess.init(args.argv, args.allocator);
 
         child.stdin_behavior = .Ignore;
         // child.stdout_behavior = .Pipe;

--- a/shader_compiler/shader_compiler.zig
+++ b/shader_compiler/shader_compiler.zig
@@ -2,7 +2,7 @@ const std = @import("std");
 const path = std.fs.path;
 const Builder = std.build.Builder;
 const Step = std.build.Step;
-const BufMap = std.BufMap;
+const EnvMap = std.process.EnvMap;
 
 const ShdcParser = @import("shdc_parser.zig").ShdcParser;
 const ShaderProgram = @import("shdc_parser.zig").ShaderProgram;
@@ -373,7 +373,7 @@ pub const ShaderCompileStep = struct {
         argv: []const []const u8,
         cwd: ?[]const u8 = null,
         cwd_dir: ?std.fs.Dir = null,
-        env_map: ?*const BufMap = null,
+        env_map: ?*EnvMap = null,
         max_output_bytes: usize = 50 * 1024,
         expand_arg0: std.ChildProcess.Arg0Expand = .no_expand,
     }) !std.ChildProcess.ExecResult {

